katran/lib/KatranLb.h

```diff
@@ -477,6 +477,16 @@ class KatranLb {
    */
   KatranMonitorStats getKatranMonitorStats();
 
+  /**
+   * @return KatranLbStats generic stats about userspace part of katran
+   *
+   * helper function which helps to introspect internals of katran's
+   * userspace counterpart
+   */
+  KatranLbStats getKatranLbStats() {
```

Comment:
const?

Reply:
sure

Reply:
actually no. in a way that all other methods returns non const value. imo sep diff required to change this behavior everywhere. i dont like to add one off in this one. or touch other non related functions